js/deer-record.js

```diff
@@ -470,6 +483,9 @@ export default class DeerReport {
 /**
  * Generate a new object URI for a resource. Abstract additional
  * properties to annotations.
+
+ * Note: As of 8/14/2020, this appears to be unused.
```

Comment:
It is definitely not. It refers to things that are no longer in the constructor. Safe to remove in this pull.

Reply:
Ok, will remove.